roles/operator_scorecard_tests/tasks/main.yml

```diff
@@ -35,6 +35,7 @@
   shell: "{{ oc_bin_path }} delete -f {{ cr_path }} --ignore-not-found=true --grace-period=60 --timeout=120s"
   environment:
     KUBECONFIG: "{{ kubeconfig_path }}"
+  ignore_errors: true
```

Comment:
Please add the comment why do we ignore errors here.
Neither code nor commit message explain the reason. I was able to find reason in different discussion

Reply:
@lslebodn Comment added